rotkehlchen/data_import/importers/bitstamp.py

```diff
@@ -0,0 +1,160 @@
+
+from rotkehlchen.accounting.structures.balance import Balance
+from rotkehlchen.accounting.structures.base import HistoryEvent
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+from rotkehlchen.assets.converters import asset_from_bitstamp
+from rotkehlchen.constants import ZERO
+from rotkehlchen.data_import.importers.constants import BITSTAMP_EVENT_PREFIX
+from rotkehlchen.data_import.utils import BaseExchangeImporter
+from rotkehlchen.db.drivers.gevent import DBCursor
+from rotkehlchen.errors.asset import UnknownAsset
+from rotkehlchen.errors.misc import InputError
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.serialization.deserialize import (
+    deserialize_asset_amount,
+    deserialize_fee,
+    deserialize_timestamp_from_date,
+)
+from rotkehlchen.types import Location, TradeType
+from rotkehlchen.utils.misc import ts_sec_to_ms
+
+
+class BitstampTransactionsImporter(BaseExchangeImporter):
+    def _consume_bitstamp_transaction(
+            self,
+            write_cursor: DBCursor,
+            csv_row: dict[str, Any],
+            timestamp_format: str = '%b. %d, %Y, %I:%M %p',
+    ) -> None:
+        """
+        Consume the file containing only trades from Bitstamp.
+        - UnknownAsset
+        - DeserializationError
+        - KeyError
+        - ValueError
+        """
+        timestamp = ts_sec_to_ms(deserialize_timestamp_from_date(
+            date=csv_row['Datetime'],
+            formatstr=timestamp_format,
+            location='Bitstamp',
+        ))
+
+        amount, amount_symbol = csv_row['Amount'].split(' ')
+        event_identifier = f'{BITSTAMP_EVENT_PREFIX}_{uuid4().hex}'
+
+        if csv_row['Type'] == 'Market':
+            value_amount, value_symbol = csv_row['Value'].split(' ')
+            fee_amount, fee_symbol = csv_row['Fee'].split(' ')
+            trade_type = TradeType.deserialize(csv_row['Sub Type'])
+
+            # if trade_type buy
+            bought_amount = deserialize_asset_amount(amount)
+            sold_amount = deserialize_asset_amount(value_amount)
+            bought_currency = asset_from_bitstamp(amount_symbol)
+            sold_currency = asset_from_bitstamp(value_symbol)
+
+            if trade_type == TradeType.SELL:
+                bought_amount, sold_amount = sold_amount, bought_amount
+                sold_currency, bought_currency = bought_currency, sold_currency
+
+            fee_amount = deserialize_fee(fee_amount)
+            fee_currency = asset_from_bitstamp(fee_symbol)
+
+            spend_trade_event = HistoryEvent(
+                event_identifier=event_identifier,
+                sequence_index=0,
+                timestamp=timestamp,
+                location=Location.BITSTAMP,
+                asset=sold_currency,
+                balance=Balance(
+                    amount=sold_amount,
+                    usd_value=ZERO,
+                ),
+                notes='',
```

Comment:
Could you add a note to this events? if you open the app and navigate to history events you will see that there is redeable message with each event. Notes are important for the user to know what is happening.

Here you have an example of how we format them for evm events https://github.com/rotki/rotki/blob/7f4165af0c7ad43431e522345a36c244343bacd6/rotkehlchen/tests/unit/decoders/test_curve.py#L1216